code/src/java/pcgen/gui2/tabs/PurchaseInfoTab.java

```diff
@@ -632,14 +621,14 @@ public void install()
 
 	}
 
-	private class RemoveAction extends AbstractAction
+	private final class SellAction extends AbstractAction
 	{
 
 		private final CharacterFacade character;
 
-		public RemoveAction(CharacterFacade character)
+		private SellAction(CharacterFacade character)
 		{
-			super(LanguageBundle.getString("in_ieRemEq")); //$NON-NLS-1$
```

Comment:
Removing the comment seems not on purpose.

Reply:
semi-intentional. I'd prefer we use `@SuppressWarnings` instead. That said, I'm separating the baseline cleanup from the adding of the button.